docker/docker_nightly.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+import os
+
+def get_nightly_version():
+    today = date.today()
+    return today.strftime("%Y.%m.%d")
+
+if __name__ == "__main__":
+    project = "torchserve-nightly"
+    cpu_version = f"{project}:cpu-{get_nightly_version()}"
+    gpu_version = f"{project}:gpu-{get_nightly_version()}"
+
+    # Build Nightly images and append the date in the name
+    os.system(f"./build_image.sh -bt dev -t pytorch/{cpu_version}")
+    os.system(f"./build_image.sh -bt dev -g -cv cu102 -t pytorch/{gpu_version}")
```

Comment:
Is there a reason we're building this with cu102 and not cu113?

Reply:
Our experience is generally it's been more stable but if that's no longer the case if we just update this line to `cu1113` and the nightly builds will work just fine